wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -75,7 +76,10 @@ def __init__(
         )
         # Make sure the provided sweep_id corresponds to a valid sweep
         try:
-            self._api.sweep(sweep_id, "{}", entity=self._entity, project=self._project)
+            resp = self._api.sweep(
```

Comment:
this handling will have to be added for the external scheduler path anyway, I don't see a harm adding this now